lib/polyfill/patchedmediakeys_webkit.js

```diff
@@ -702,8 +727,11 @@ shaka.polyfill.PatchedMediaKeysWebkit.MediaKeySession.prototype.generate_ =
   // GKR can throw an exception.  If this occurs, wait 10 ms and try again at
   // most once.  This situation should only occur when init data is available
   // ahead of the 'needkey' event.
+
+  var prefixApi = shaka.polyfill.PatchedMediaKeysWebkit.prefixApi;
+  var prefixedApi = prefixApi('generateKeyRequest');
```

Comment:
Having an alias variable called "prefixApi" and another variable called "prefixedApi" is a little confusing and could easily lead to typos.  Please rename this to "generateKeyRequestName", which will both avoid confusion, describe which API it references, and indicate that it is a string, not a function.